src/managers/KeybindManager.cpp

```diff
@@ -424,7 +441,7 @@ int repeatKeyHandler(void* data) {
     return 0;
 }
 
-bool CKeybindManager::handleKeybinds(const uint32_t& modmask, const std::string& key, const xkb_keysym_t& keysym, const int& keycode, bool pressed, uint32_t time) {
+bool CKeybindManager::handleKeybinds(const uint32_t modmask, const std::string& key, const SPressedKeyWithMods keyNumeric, bool pressed, uint32_t time) {
```

Comment:
since this is const yu can avoid copying with `const SPressedKeyWithMods&`

Reply:
Sure thing chief